doc/source/configuration/magnum-capi.rst

```diff
@@ -0,0 +1,36 @@
+========================
+Magnum Cluster API Driver
+========================
+
+Prequisites for deploying the CAPI driver in magnum: 
+
+Management Cluster
+==================
+The CAPI driver relies on a management cluster to manage tenant kubernetes clusters. 
+The easiest way to get one is by deploying [this](https://github.com/stackhpc/azimuth-config/tree/feature/capi-mgmt-config) branch of azimuth-config, and look at the `capi-mgmt-example` environment. 
+
+
+Ensure that you have set `capi_cluster_apiserver_floating_ip: true`, as the management cluster will need an externally accessible IP.
```

Comment:
Can we define external here please? Who need to talk to the end points we have here? Maybe reference the main azimuth docs here?